xml/SkinSettings.xml

```diff
@@ -77,6 +77,12 @@
 					<onclick>Skin.ToggleSetting(no_fanart)</onclick>
 					<selected>!Skin.HasSetting(no_fanart)</selected>
 				</control>
+				<control type="radiobutton" id="606">
+					<label>$LOCALIZE[31163]</label>
+					<include>DefaultSettingButton</include>
+					<onclick>Skin.ToggleSetting(hide_background_fanart)</onclick>
+					<selected>!Skin.HasSetting(hide_background_fanart)</selected>
+				</control>
 				<control type="button" id="607">
```

Comment:
Yep, anyway I tried to include this setting into Custom_1105_MusicOSDSettings.xml in the form:

Show Fanart background: Yes

or 

Show Fanart background: No

with onclick Skin.ToggleSetting. It works, but I have not been able to display a Yes/No label depending on the skin setting boolean value.